models/order/src/delivery/transformers.ts

```diff
@@ -0,0 +1,19 @@
+import { Transformer } from '@commercetools-test-data/core';
+import type { TDelivery, TDeliveryGraphql } from './types';
+
+const transformers = {
+  default: Transformer<TDelivery, TDelivery>('default', {
+    buildFields: ['items', 'parcels', 'address'],
```

Comment:
I think `custom` should be added here, incase if the customfield is implemented then that model has to be built. Similar to how it is written in below  `models/order/src/parcel/transformers.ts`

Same for other transformers below like `models/order/src/line-item-return-item/transformers.ts`, etc

Reply:
@jaikumar-tj Please see [this comment](https://github.com/commercetools/test-data/pull/669#discussion_r1745521489) below.